server/src/item.rs

```diff
@@ -14,11 +14,17 @@ use zbus::zvariant::OwnedObjectPath;
 
 use crate::{Service, collection::Collection, error::custom_service_error};
 
+#[derive(Debug, Clone)]
+enum InnerItem {
+    Locked(file::LockedItem),
+    Unlocked(file::Item),
+}
```

Comment:
Should we have this enum in client/file and rename `file::Item` to `file::UnlockedItem` or so?

Reply:
If so, we should do the same for the keyring type.

Reply:
Renamed both in the latest commits.